dataprofiler/profilers/profiler_options.py

```diff
@@ -106,7 +106,8 @@ def validate(self, raise_error=True):
         errors = self._validate_helper()
         if raise_error and errors:
             raise ValueError('\n'.join(errors))
-        return errors
+        elif errors: 
+            return errors
```

Comment:
@JGSweets modified to return None on validate with tests changed accordingly

Reply:
ty